Makefile

```diff
@@ -226,19 +226,8 @@ update-po:: $(PO_TEMPLATE)
 $(ALL_MSGFILES): %.msg : %.po
 	$(QUIET_MSGFMT0)$(MSGFMT) --statistics --tcl -l $(basename $(notdir $<)) -d $(dir $@) $< $(QUIET_MSGFMT1)
 
-lib/tclIndex: $(ALL_LIBFILES) GIT-GUI-BUILD-OPTIONS
-	$(QUIET_INDEX)if echo \
-	  $(foreach p,$(PRELOAD_FILES),source $p\;) \
-	  auto_mkindex lib $(patsubst lib/%,%,$(sort $(ALL_LIBFILES))) \
-	| $(TCL_PATH) $(QUIET_2DEVNULL); then : ok; \
```

Comment:
This removal obsoletes Makefile variable `QUIET_2DEVNULL`.

Reply:
Good catch, fixed.